### ISSUE TITLE
Links and images

### ISSUE THREAD

**Issue by gbelvedere** [ageitgey/node-unfluff#90]
Is it possible to get an array of the images that are embeded in the content and their positions? and same thing with the links?

**Comment by ageitgey** [comment]
No, that's not currently a feature. It just returns one "main" image.

**Comment by gbelvedere** [comment]
any idea how much work would be to implement it?  is it possible to collaborate with the project and do it?
